<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddContactDiag</class>
 <widget class="QDialog" name="AddContactDiag">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>163</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add contact</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="1,7,1">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4" stretch="2,4">
       <property name="topMargin">
        <number>6</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <property name="leftMargin">
          <number>5</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <item alignment="Qt::AlignBottom">
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <family>Tahoma</family>
             <pointsize>9</pointsize>
             <italic>true</italic>
            </font>
           </property>
           <property name="text">
            <string>Search :</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <property name="leftMargin">
          <number>5</number>
         </property>
         <property name="rightMargin">
          <number>5</number>
         </property>
         <item alignment="Qt::AlignTop">
          <widget class="QLineEdit" name="SearchNameField">
           <property name="font">
            <font>
             <family>Tahoma</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>Search someone..</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3" stretch="2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item alignment="Qt::AlignTop">
          <widget class="QPushButton" name="AddContactButton">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>Add selected contact</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="CloseButton">
         <property name="minimumSize">
          <size>
           <width>110</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>110</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Tahoma</family>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Close</string>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
